pkg/httputil/logging.go

```diff
@@ -76,9 +75,10 @@ func DefaultLoggingMiddleware(requestIDHeaderName string, fields logging.Fields,
 			next.ServeHTTP(writer, r) // handle the request
 
 			loggingFields := logging.Fields{
-				"took":        time.Since(startTime),
-				"status_code": writer.StatusCode,
-				"sent_bytes":  writer.ResponseSize,
+				"took":           time.Since(startTime),
+				"status_code":    writer.StatusCode,
+				"sent_bytes":     writer.ResponseSize,
+				logging.LogAudit: "API",
```

Comment:
I don't think we need this one, at least not here - the same middleware is used for logging s3 and api requests.
The field "service_name" value can be "s3_gateway" or "rest_api" to capture which instance of this middleware we report from.
If service_name is used more than this middleware we can mark this log with a different label to separate it from other logging lines - but it is not just API.